File: edad.cpp
#include<iostream>
using namespace std;
int main() {
	int edad;
	edad = 0;
	cout << "Ingresa tu edad: " << endl;
	cin >> edad;
	if (edad>=18) {
		cout << "Estas listo para votar" << endl;
	} else {
		cout << "No puedes votar" << endl;
	}
	return 0;
}

File: edad.java
import java.util.Scanner;

public class edad {

	public static void main(String args[]) {
    Scanner s = new Scanner(System.in);
		
		System.out.println("Ingresa tu edad:");
		int edad = s.nextInt();
		if (edad>=18) {
			System.out.println("!Estas listo para votar¡");
		} else {
			System.out.println("!No puedes votar¡¡¡");
		}
	}
}